GUI/QtQuick/QtQuickAdapter.h

```diff
@@ -211,6 +212,8 @@ class QtQuickAdapter : public QObject, public ApiBusAdapter, public bs::MainLoop
    void failedDeleteWallet();
    void successDeleteWallet();
    void walletSeedAuthFailed();
+   void transactionExported(const QString&);
```

Comment:
Please put the arg name here, otherwise it's no explicitly clear what the `QString` is for. What exactly is supposed to be passed here?